operator/pkg/marshall_unmarshall_test.go

```diff
@@ -0,0 +1,33 @@
+package operator
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ugorji/go/codec"
+)
+
+// Test roundtrip of cbor serialization used in Aligned.
+func FuzzMarshalUnmarshal(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte, seed int64) {
+		// MarshalUnmarshal
+
+		var marshalled VerificationData
+		decoder := codec.NewDecoderBytes(data, new(codec.CborHandle))
+		err := decoder.Decode(marshalled)
```

Comment:
[Decode() must be passed a pointer](https://github.com/ugorji/go/blob/master/codec/decode.go#L1492)
```suggestion
		err := decoder.Decode(&marshalled)
```